docs/releases/v1.15.4-2.md

```diff
@@ -0,0 +1,262 @@
+# Release notes
+
+- [Release notes](#release-notes)
+  - [Auto-join nodes feature](#auto-join-nodes-feature)
+    - [Use case](#use-case)
+    - [Issue](#issue)
+    - [Change](#change)
+      - [How to deploy it](#how-to-deploy-it)
+        - [Before start](#before-start)
+        - [Requirements](#requirements)
+        - [Upgrading an existing cluster](#upgrading-an-existing-cluster)
+          - [Terraform part](#terraform-part)
+          - [Ansible part](#ansible-part)
+        - [Verify](#verify)
+
+This version contains a new feature: **auto-join** nodes.
+
+## Auto-join nodes feature
+
+### Use case
+
+Having a kubernetes cluster with nodes belonging to an autoscaling group. Node pool has to be able to create and destroy in a dynamic way nodes. New nodes needs to join the kubernetes cluster without human intervention.
+
+### Issue
+
+Before this feature, an operator had to run an ansible playbook every time a new node was created to join the cluster. This causes some problems in cloud environments. You ussually deploy’s your nodes in an autoscalling group. This means that your node pool can grow *(positive and negative)* the number of nodes it has. It happens in a very dynamic way, it can cause having instances not belonging to the kubernetes cluster. It’s not only a matter of scale, when cloud providers needs to do some maintainning stuff, they relies in the autoscalling group killing an instance and letting the autoscalling group to recreate it.
```

Comment:
Typo: "ussually"
Typo: "deploy's" -> "deploy".
Rephrase:  "This means that your node pool can grow *(positive and negative)* the number of nodes it has." -> "This means that the number of nodes in a given pool can increase or decrease".
Typo: "maintainning" -> "maintenance"
Typo: "relies" -> "rely"
Typo: "autoscalling" -> "autoscaling"